spec/requests/post_controller_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+RSpec.describe PostsController, type: :request do
+  describe 'Check post main page' do
+    before(:example) do
+      get '/users/1/posts'
+      expect(response).to render_template(:index)
+    end
+
+    it 'Checks if the status response is succes(OK)' do
+      expect(response.status).to be(200)
+    end
+
+    it 'Check the if the rendered template is correct' do
+      expect(response.body).to include('<p>this is a list of all posts</p>')
+    end
+  end
+
+  describe 'Check posts for specific user' do
+    before(:example) do
+      get '//users/1/posts/:id'
+      expect(response).to render_template(:show)
```

Comment:
- [ ] The `expect` should be in its own **`it`** block. Kindly remove it from here.

Reply:
I will fix that immediately, thanks.